test/format/ops.c

```diff
@@ -416,12 +416,10 @@ operations(u_int ops_seconds, u_int run_current, u_int run_total)
         if (!running)
             break;
         __wt_sleep(0, 250 * WT_THOUSAND); /* 1/4th of a second */
-
-        if (fourths == 1 && GV(RUNS_PREDICTABLE_REPLAY))
-            replay_end_timed_run();
```

Comment:
If we're thinking of merging this into develop sometime, we'll need to fix this change for predictable replay (the call to `replay_end_timed_run()` got lost).  Alternatively, these changes could be made with `#ifdef ENABLE_ANTITHESIS`